src/apps/reservation-list/modal/reservation-details/helper.ts

```diff
@@ -0,0 +1,34 @@
+import { ComplexSearchWithPaginationWorkAccessQuery } from "../../../../core/dbc-gateway/generated/graphql";
+
+type AccessManifestations =
+  ComplexSearchWithPaginationWorkAccessQuery["complexSearch"]["works"][0]["manifestations"]["all"];
+
+export const findAccessManifestationByIdentifier = (
+  manifestations: AccessManifestations,
+  identifier: string
+) => {
+  return manifestations.filter((manifestation) =>
+    manifestation.identifiers.find(
+      (id) => id.value.replace(/\D/g, "") === identifier.replace(/\D/g, "")
+    )
+  );
+};
+
+export const findEreolAccessLinkFromManifestations = (
+  manifestations: AccessManifestations
+) => {
+  const manifestationWithEreolAccess = manifestations.find((manifestation) => {
+    return manifestation.access.find((access) => {
+      return access.__typename === "Ereol";
```

Comment:
Dont we have a definition for the `Ereol` string somewhere instead of that we define a "random" string here.

Reply:
Nopesies.